src/forms/FormFieldAdvancedSelection/FormFieldAdvancedSelection.tsx

```diff
@@ -90,7 +90,7 @@ const FormFieldAdvancedSelection = (props: MosaicFieldProps<AdvancedSelectionDef
 					<Button
 						color="teal"
 						variant="text"
-						label="Add Element"
+						label="Add"

```

Comment:
The update to the component looks good... but the unit tests should also be updated. Please whenever working on a ticket we need to ensure that whatever we changed has its corresponding tests updated.